### PR TITLE
Orbit PyTest Regression tests.

### PR DESCRIPTION
**Pull request by stthoom**
### Description:

**Replace with description of the changes**

### Issue Reference URL

https://github.com/awslabs/aws-orbit-workbench/issues/<NUMBER>

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

File: test/regressions/scripts/pytest-lake-creator-multithreaded.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+pytest --kube-config /Users/stthoom/.kube/config -v -s  -k testlakecreator_zip -n auto
+
+pytest --kube-config /Users/stthoom/.kube/config -v -s  -k testlakecreator_unzip_check
+
+# Multi threaded
+pytest --kube-config /Users/stthoom/.kube/config -v -s  -k testlakecreator_glue -n auto
+
+pytest --kube-config /Users/stthoom/.kube/config -v -s  -k testlakecreator_checkglue
+
+pytest --kube-config /Users/stthoom/.kube/config -v -s  -k testlakecreator_lf
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-creator-multithreaded.sh:22]: Can you remove your username?

File: test/regressions/scripts/pytest-lake-admin.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+# Required os env variables. Replace with testing Orbit env details
+export AWS_ORBIT_ENV=iter
+export AWS_ORBIT_TEAM_SPACE=lake-admin
+
+# Set the .kube/config with respect to runtime environment
+pytest --kube-config /Users/stthoom/.kube/config -v -s  -k testlakeadmin -n auto
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-admin.sh:10]: Can you remove your username?

File: test/regressions/scripts/pytest-lake-user.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+# Required os env variables. Replace with testing Orbit env details
+export AWS_ORBIT_ENV=iter
+export AWS_ORBIT_TEAM_SPACE=lake-user
+
+# Set the .kube/config with respect to runtime environment
+pytest --kube-config /Users/stthoom/.kube/config -v -s  -k testlakeuser -n auto
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-user.sh:10]: Can you remove your username?

File: test/regressions/scripts/pytest-lake-creator.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+# Required os env variables. Replace with testing Orbit env details
+export AWS_ORBIT_ENV=iter
+export AWS_ORBIT_TEAM_SPACE=lake-creator
+
+# Set the .kube/config with respect to runtime environment
+pytest --kube-config /Users/stthoom/.kube/config -v -s  -k lakecreator
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-creator.sh:10]: Can you remove your username?

File: test/regressions/scripts/pytest-lake-creator.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+# Required os env variables. Replace with testing Orbit env details
+export AWS_ORBIT_ENV=iter
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-creator.sh:6]: the env is hardcoded

File: test/regressions/scripts/pytest-lake-creator.sh [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+export AWS_ORBIT_TEAM_SPACE=lake-creator
+export AWS_ORBIT_KUBE_CONFIG=
+
+# Set the .kube/config with respect to runtime environment
+pytest --kube-config ~/.kube/config -v -s  -k testlakecreator
+
+
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-creator.sh:13]: The env is hardcoded...can it be pulled from session / environment params?

File: test/regressions/scripts/pytest-lake-user.sh [hunk 7]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+# Required os env variables. Replace with testing Orbit env details
+export AWS_ORBIT_ENV=iter
+export AWS_ORBIT_TEAM_SPACE=lake-user
+
+# Set the .kube/config with respect to runtime environment
+pytest --kube-config ~/.kube/config -v -s  -k testlakeuser -n auto
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-user.sh:10]: The env is hardcoded...can it be pulled from session / environment params?

File: test/regressions/scripts/pytest-lake-admin.sh [hunk 8]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+# Required os env variables. Replace with testing Orbit env details
+export AWS_ORBIT_ENV=iter
+export AWS_ORBIT_TEAM_SPACE=lake-admin
+
+# Set the .kube/config with respect to runtime environment
+pytest --kube-config ~/.kube/config -v -s  -k testlakeadmin -n auto
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-admin.sh:10]: The env is hardcoded...can it be pulled from session / environment params?

File: test/regressions/scripts/pytest-lake-creator-multithreaded.sh [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+pytest --kube-config ~/.kube/config -v -s  -k testlakecreator_zip -n auto
+
+pytest --kube-config ~/.kube/config -v -s  -k testlakecreator_unzip_check
+
+# Multi threaded
+pytest --kube-config ~/.kube/config -v -s  -k testlakecreator_glue -n auto
+
+pytest --kube-config ~/.kube/config -v -s  -k testlakecreator_checkglue
+
+pytest --kube-config ~/.kube/config -v -s  -k testlakecreator_lf
```
Comment (dgraeber) [inline test/regressions/scripts/pytest-lake-creator-multithreaded.sh:22]: The env is hardcoded...can it be pulled from session / environment params?

**Review by chamcca — APPROVED** [review]

**Pull request merged**